### PR TITLE
Cs fixer

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: _tuts/2-structured-data.diff [hunk 1]
```diff
@@ -3184,76 +3183,6 @@ index 0000000..d2f9240
...
--
--namespace Google\Cloud\Samples\Bookshelf;
--
--use Behat\Mink\Driver\GoutteDriver;
--use Behat\Mink\Session;
--
--/**
-- * Class BookshelfTest
-- */
--class BookshelfTest extends \PHPUnit_Framework_TestCase
```
Comment (bshaffer) [inline _tuts/2-structured-data.diff]: Are these changes expected? Essentially we are removing `BookshelfTest` and adding `E2eTest`...

I feel like this is a bug / error.

**Comment by bshaffer** [comment]
This looks great! What about adding a `.php_cs` file?

**Comment by tmatsuo** [comment]
Yeah feel free to add `.php_cs`.
I'd like to merge this to fix our jenkins build.

**Comment by bshaffer** [comment]
ok, then LGTM

**Pull request merged**